app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -281,6 +267,11 @@ class BrowserFragment :
             owner = this,
             view = browserToolbar
         )
+
+        // sets the background of the URL when already on a page
+        browserToolbar.display.setUrlBackground(
```

Comment:
This shouldn't happen when we're in a Custom Tab. You probably want to put this in `initialiseNormalBrowserUi` where a few lines above you can see that we decorate the UI based on a custom tab check.